### DISCUSSION TITLE
Copilot Chat enabled but for not available in IntelliJ

### DISCUSSION THREAD

**Discussion by hagaic** [community/community#75510] [Copilot Conversations]
### Select Topic Area

Bug

### Body

As an admin, I enabled GitHub Copilot Chat for IDEs, but the chat window is not available in IntelliJ.

**Comment by Akash1134** [comment]
👋  Hi @hagaic ,

If you are an owner of an invoiced organization or enterprise with a Copilot Business subscription and would like to participate in the beta for [Copilot Chat in a JetBrains IDE](https://github.com/orgs/community/discussions/69250), you can contact your account manager or solutions engineer to request access. Please note that customers on Free, Team or self-serve Enterprise plans do not have an account manager and are not eligible for the private beta at this time.

Thanks!
